src/main/java/com/scalar/database/transaction/consensuscommit/CommitMutationComposer.java

```diff
@@ -19,10 +19,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * Keep track of mutations that need to be committed to {@link
```

Comment:
I think it is confusing. These mutations change the state of records to commit state.

Reply:
That is confusing! Thank you. Fixed.